sw/device/rom_exts/docs/manifest.md

```diff
@@ -121,7 +143,7 @@ Notes:
 
     Reserved fields have fixed size and unspecified alignment.
 
-## Field Descriptions
```

Comment:
I think it would have been okay to just keep the header order as the title is the main "header".
This is no rule, but would be more in alignment to the other documents (but not all), it depends on what is defined as a major section.

Reply:
I would say these are major sections, now the title is separate.